src/components/MoneyRequestConfirmationList.js

```diff
@@ -132,6 +132,7 @@ function MoneyRequestConfirmationList(props) {
 
     const selectedParticipants = useMemo(() => _.filter(props.participants, (participant) => participant.selected), [props.participants]);
     const payeePersonalDetails = useMemo(() => props.payeePersonalDetails || props.currentUserPersonalDetails, [props.payeePersonalDetails, props.currentUserPersonalDetails]);
+    const canModifyParticipants = !props.isReadOnly && props.canModifyParticipants && props.hasMultipleParticipants;
```

Comment:
Can we have a better naming here `canModifyParticipants` off as we are not modifying who paid anyway?

Reply:
Okayish make the changes, thanks!